src/features/import-maps.js

```diff
@@ -53,6 +53,14 @@ function processScripts () {
       }).catch(function (err) {
         err.message = errMsg('W4', process.env.SYSTEM_PRODUCTION ? script.src : 'Error fetching systemjs-import map ' + script.src) + '\n' + err.message;
         console.warn(err);
+        if (typeof script.onerror === 'function') {
+          try {
+            script.onerror();
+          } catch(callbackErr) {
+            callbackErr.message = 'Error during onerror script for systemjs-importmap: ' + script.src + '\n' + callbackErr.message;
```

Comment:
This error should be obtainable from the stack anyway, so let's remove this try-catch wrapper rather. Or did you specifically find the stack didn't have the information needed?

Reply:
The stack has the required info, I will remove the try-catch, thanks for the quick review 👍